examples/codebook/codebook.json

```diff
@@ -1,24 +1,37 @@
-[
-  {
-    "codeword": [
-      {"r": 0, "c": 0, "v": 1},
-      {"r": 0, "c": 1, "v": 1},
-      {"r": 1, "c": 1, "v": 1}
-    ],
-    "target": "SCUBE2"
-  },
-  {
-    "codeword": [
-      {"r": 0, "c": 0, "v": 1},
-      {"r": 1, "c": 1, "v": 1}
-    ],
-    "target": "BRCA"
-  },
-  {
-    "codeword": [
-      {"r": 0, "c": 1, "v": 1},
-      {"r": 1, "c": 0, "v": 1}
-    ],
-    "target": "ACTB"
-  }
-]
\ No newline at end of file
+{
+  "codes": [
+    {
+      "codeword": [
+        {"r": 0, "c": 0, "v": 1},
+        {"r": 0, "c": 1, "v": 1},
+        {"r": 1, "c": 1, "v": 1}
+      ],
+      "target": "SCUBE2"
+    },
+    {
+      "codeword": [
+        {"r": 0, "c": 0, "v": 1},
+        {"r": 1, "c": 1, "v": 1}
+      ],
+      "target": "BRCA"
+    },
+    {
+      "codeword": [
+        {"r": 0, "c": 1, "v": 1},
+        {"r": 1, "c": 0, "v": 1}
+      ],
+      "target": "ACTB"
+    }
+  ],
+  "channel_map": [
```

Comment:
why not do a map of channel name to index val?  at least you gain channel name uniqueness as a freebie.

Reply:
I could also see having it as an array. cF. https://github.com/IDR/idr-metadata/blob/403b0817c1c57dd6b509cfb0d0d41bd35ec1b938/idr0006-fong-nuclearbodies/screenA/idr0006-screenA-renderdef.yml

```
1:
  label: DAPI
  min: 0
  max: 1000
  color: "0000FF"
2:
  label: TRITC
  min: 50,
  max: 800
```
since an ordering is guaranteed.